src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Validators/PhoneNumberAttributeTests.cs

```diff
@@ -11,6 +11,8 @@ public class PhoneNumberAttributeTests
     [InlineData("01234567")]
     [InlineData("0   123 4567")]
     [InlineData("0 1 2 3   4 5 6 7      ")]
+    [InlineData("")]
+    [InlineData("    ")]
```

Comment:
I tried adding this as a test case which I think should fail but it passes:

`    [InlineData("0 1 2 3   4 5 6        ")]`

Reply:
That should pass. The minimum characters includes spaces. I didn't consider it worth the effort to cover that case.

Reply:
I appreciate the phone number might be formatted with spaces like `01234 567890` or `0123 456 789`. I'm just a bit surprised that you can pass the validation by appending whitespace.

These would have been better examples than in my original comment of passing test cases which I think we would expect to fail:

```
[InlineData("0       ")]
[InlineData("0123456 ")]
```

Reply:
Those would fail. It's whitespace between characters that causes an issue. So '0                0' passes when you would probably expect it to fail.

Reply:
I only left this comment because they were passing when I tried them. If they fail now after recent changes, that's good. 🙂

Reply:
Okay - actually you're correct - those will pass on the backend. I'll add a fix for that.

Reply:
I've pushed a fix.